Adjust Satel Integra for config and subentry flow (#37609)
home-assistant/home-assistant.github.io: source/_integrations/satel_integra.markdown

For the binary sensor, check the [type/class](/integrations/binary_sensor/) list for a possible visualization of your zones. Note: If no zones or outputs are specified, Home Assistant will not load any binary_sensor integrations."

{% include integrations/config_flow.md %}

{% configuration_basic %}
Host:
  description: The IP address of the Satel Integra ETHM module.
Port:
  description: The port on which the ETHM module listens for clients using integration protocol.
Code:
  description: Optional code that will be used to toggle switchable outputs.
{% endconfiguration_basic %}

## Settings

> After setting up the connection details, partitions, zones and outputs can be configured using the <i>Options</i> on the <i>Satel Integra</i> card on the {% my integrations title="Settings > Devices & services" %} page.

Reviewer: ```suggestion
After setting up the connection details, partitions, zones, and outputs can be configured using the **Options** on the **Satel Integra** card on the {% my integrations title="**Settings** > **Devices & services**" %} page.
```
we use bold to mark UI strings

> ### Partitions, Zones and Outputs

Reviewer: ```suggestion
### Partitions, zones, and outputs
```
we use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization) also in headings

> Partitions, Zones and outputs can be added, edited, and removed through the option forms.

Reviewer: ```suggestion
Partitions, zones, and outputs can be added, edited, and removed through the option forms.
```

> Each partition will have it's own alarm panel, each zone and output will have a binary sensor and each switchable output will have a switch created.

Reviewer: ```suggestion
Each partition will have its own alarm panel, each zone and output will have a binary sensor, and a switch will be created for each switchable output.
```

> Each partition will have its own alarm panel, each zone and output will have a binary sensor, and a switch will be created for each switchable output.

Reviewer: ```suggestion
Each partition will have its own alarm panel. Each zone and output will have a binary sensor, and a switch will be created for each switchable output.
```

> To get started, select which entry type you want to manage:

Reviewer: ```suggestion
1. To get started, select which entry type you want to manage:
```

> ![image](https://github.com/user-attachments/assets/c788689b-9f58-46c4-9e51-d6cce5431763)

Reviewer: please upload the images into this repo, under `/images/integrations/satel_integra`

> ![Start options flow](/images/integrations/satel_integra/satel_integra_options_flow.png)

Reviewer: ```suggestion
   ![Start options flow](/images/integrations/satel_integra/satel_integra_options_flow.png)
```

> Then select the partition/zone/output number you want to manage and the corresponding action. The first action will always be to Add (unless a previous YAML configuration was imported)

Reviewer: ```suggestion
2. Then, select the partition/zone/output number you want to manage and the corresponding action. 
    - The first action will always be to **Add** (unless a previous YAML configuration was imported).
```

> ![Configure partitions](/images/integrations/satel_integra/satel_integra_partition_configuration.png)

Reviewer: ```suggestion
    ![Configure partitions](/images/integrations/satel_integra/satel_integra_partition_configuration.png)
```

> Afterwards fill in the required fields for each entry type. Saving the options form should automatically generate and/or update the {%term entities %}.

Reviewer: ```suggestion
3. Afterwards, fill in the required fields for each entry type. 
    - Saving the options form should automatically generate and/or update the {%term entities %}.
```

> Deleting an entry removes the corresponding entry from the configuration, but the entity will still exist, you will manually have to remove this from the list afterwards.

Reviewer: ```suggestion
4. Deleting an entry removes the corresponding entry from the configuration, but the entity will still exist. 
    -You will manually have to remove this entity from the list afterward.
```

Port:
  description: The port on which the ETHM module listens for clients using integration protocol.
Code:
  description: Optional code that will be used to toggle switchable outputs.
{% endconfiguration_basic %}

## Settings

After setting up the connection details, partitions, zones, and outputs can be configured using the **Options** on the **Satel Integra** card on the {% my integrations title="**Settings** > **Devices & services**" %} page.

### Partitions, zones, and outputs

Partitions, zones, and outputs can be added, edited, and removed through the option forms.

Each partition will have its own alarm panel. Each zone and output will have a binary sensor, and a switch will be created for each switchable output.

1. To get started, select which entry type you want to manage:

   ![Start options flow](/images/integrations/satel_integra/satel_integra_options_flow.png)

2. Then, select the partition/zone/output number you want to manage and the corresponding action. 
    - The first action will always be to **Add** (unless a previous YAML configuration was imported).

    ![Configure partitions](/images/integrations/satel_integra/satel_integra_partition_configuration.png)

> 3. Afterwards, fill in the required fields for each entry type. 

Reviewer: ```suggestion
3. Afterward, fill in the required fields for each entry type. 
```

    - Saving the options form should automatically generate and/or update the {%term entities %}.
4. Deleting an entry removes the corresponding entry from the configuration, but the entity will still exist. 

>     -You will manually have to remove this entity from the list afterward.

Reviewer: ```suggestion
    - You will manually have to remove this entity from the list afterward.
```

For the binary sensor, check the [type/class](/integrations/binary_sensor/) list for a possible visualization of your zones. Note: If no zones or outputs are specified, Home Assistant will not load any binary_sensor integrations."

{% include integrations/config_flow.md %}

{% configuration_basic %}
Host:
  description: The IP address of the Satel Integra ETHM module.
Port:

>   description: The port on which the ETHM module listens for clients using integration protocol.

Reviewer: ```suggestion
  description: The port on which the ETHM module listens for clients using the integration protocol.
```

Code:
  description: Optional code that will be used to toggle switchable outputs.
{% endconfiguration_basic %}

> ## Partitions, zones, and outputs

Reviewer: ```suggestion
## Configuring partitions, zones, and outputs
```

> After setting up the connection details, partitions, zones, and outputs can be configured as **Subentries** on the {% my integration domain="satel_integra" title="**Satel Integra**" %} integration page.

Reviewer: ```suggestion
After setting up the connection details, you can configure partitions, zones, and outputs as **Subentries** on the {% my integration domain="satel_integra" title="**Satel Integra**" %} integration page.
```

> Click the 'Add partition', 'Add zone', 'Add output' or 'Add switchable output' buttons to create the respective entities and follow the config flow to set up the individual entities.

Reviewer: ```suggestion
To create the respective entities, select the **Add partition**, **Add zone**, **Add output**, or **Add switchable output** buttons. Follow the instructions in the UI to set up the individual entities.
```

> Each partition will have its own alarm panel. Each zone and output will have a binary sensor, and a switch will be created for each switchable output.

Reviewer: ```suggestion
**Result**: Each partition will have its own alarm panel. Each zone and output will have a binary sensor, and a switch will be created for each switchable output.
```
